django/test/runner.py

```diff
@@ -366,9 +371,8 @@ def get_max_test_processes():
     """
     The maximum number of test processes when using the --parallel option.
     """
-    # The current implementation of the parallel test runner requires
-    # multiprocessing to start subprocesses with fork().
-    if multiprocessing.get_start_method() != "fork":
+    # The current implementation of the parallel test runner does not support forksever.
+    if multiprocessing.get_start_method() == "forkserver":
```

Comment:
I'd prefer to use supported methods, in case there are new once in the future:
```suggestion
    # The current implementation of the parallel test runner requires
    # multiprocessing to start subprocesses with fork() or spawn().
    if multiprocessing.get_start_method() not in {"fork", "spawn"}:
```